### DISCUSSION TITLE
A number of agents who do not appear in the OneShot competition leaderboard

### DISCUSSION THREAD

**Discussion by 177amit** [yasserfarouk/scml#102] [one-shot-qa] [answered]
Hello,

I have noticed that a number of agents do not appear in the leaderboard of OneShot competition.
They appear under "Running next" but do not appear in the leaderboard table.
Relevant Agents: Agent20123, AwesomeAgent,Agent20135.
It is important to note that although agent AwesomeAgent appears in bold (i.e., it has been modified and run in the next run), this is not the case. Already a number of runners that this agent  did not participate in the competition.

**Comment by yasserfarouk** [comment]
Thank you for noticing this issue. Will investigate it asap

**Comment by yasserfarouk** [comment]
It seems that the three agents are currently running normally. Not sure what was the glitch that prevented them from running in the previous tournament.

**Reply by 177amit** [reply]
Thank you for your answer. It seems that the agents are currently running normally (except for one agent who has not yet run - Agent20135). If it's really just a display bug and these agents are running, I'm calm.

**Comment by 177amit** [comment]
Hello @yasserfarouk ,

Following on the question, I asked you a few days ago, our agent (AgentSAS) does not currently appear in the "Leader Board" but does appear under "Running in the official competition". Can you please check if everything is OK? On the leader board, there are fewer agents than the agents who will run in the official competition.

**Comment by yasserfarouk** [comment, marked as answer]
What is on the leaderboard are agents that happened to be active agent, tested and not scored in the last tournament. As long as your agent appears in the “Running in the official competition” list, it will run in the official competition given that your team submits the academic report by the deadline. 

Sorry if I missed your earlier question.

**Reply by 177amit** [reply]
Thank you for your answer.
